.github/workflows/test.yaml

```diff
@@ -170,6 +175,57 @@ jobs:
       - name: Report
         id: report
         run: echo "outcome=$([ ${{ steps.metadataYaml.outputs.exists }} = 'true' ] && echo success || echo skip)" >> $GITHUB_OUTPUT
+  tox-lint:
+    name: Lint tox.ini
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: ${{ inputs.working-directory }}
+    outputs:
+      outcome: ${{ steps.report.outputs.outcome }}
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run lint
+        run: |
+          # Check dependencies
+          EXPECTED_LINT_DEPS="\
+              mypy \
+              isort \
+              black \
+              flake8-docstrings \
+              flake8-docstrings-complete \
+              flake8-builtins \
+              flake8-test-docs \
+              pep8-naming \
+              codespell \
+              pylint \
+              pydocstyle \
+              "
+          for EXPECTED_LINT_DEP in $EXPECTED_LINT_DEPS; do
+              DEP_REGEX="deps.*=.*\n(\s.*\n)*(\s.*$EXPECTED_LINT_DEP)"
+              python3 -c "from pathlib import Path; import re; import sys; sys.exit(1) if not re.search(r'$DEP_REGEX', Path('tox.ini').read_text()) else sys.exit(0)" || \
+                  (echo "$EXPECTED_LINT_DEP should be in deps of [testenv:lint] environment in tox.ini" 1>&2 && false)
+          done
+
+          # Check cmmands
+          EXPECTED_LINT_CMDS="\
+              pydocstyle \
+              codespell \
+              flake8 \
+              isort \
+              black \
+              mypy \
```

Comment:
This is more a question than a remark: why do we include mypy in a "lint" action instead of having a separate "static analysis" one ?

Reply:
There is a static analysis section, I wonder what the difference between a linter and static analysis would be?

Reply:
In my mind, a linter was more about codestyle. But that means that a linter is a static analysis tool so nevermind